core/src/main/scala/org/apache/spark/sql/rapids/tool/store/SQLPlanModelWithDSCaching.scala

```diff
@@ -20,7 +20,7 @@ import scala.collection.mutable.ArrayBuffer
 
 /**
  * An extension of SQLPlanModel to only cache the DataSourceRecords from previous plan.
- * @param sqlId the executionID of the sqlPlan.
+ * @param sqlID the executionID of the sqlPlan.
```

Comment:
nit: The arg name is `sqlId`. We should probably revert it.
```suggestion
 * @param sqlId the executionID of the sqlPlan.
```